### ISSUE TITLE
Taint Analysis: view *name* sinks should be `include` only, not `file` (double report on one call)

### ISSUE THREAD

**Issue by alies-dev** [psalm/laravel-psalm-plugin#1353]
**Describe the bug**

A tainted view *name* is reported twice, as both `TaintedFile` and `TaintedInclude`, on a single `view()` / `View::make()` call. Only `include` is accurate for a view name: the finder cannot be steered to an arbitrary file, but it can be steered to an arbitrary template that then gets compiled and executed.

`Illuminate\View\FileViewFinder::getPossibleViewFiles()` is the reason:

```php
return array_map(fn ($extension) => str_replace('.', '/', $name).'.'.$extension, $this->extensions);
```

- every `.` in the name becomes `/`, so `../` traversal is destroyed before the path is built;
- one of `blade.php`, `php`, `css`, `html` is always appended;
- the result is only ever joined onto a configured view path (`findInPaths()`).

So a tainted name cannot produce an arbitrary file read, which is what `file` means. What it can do is select which template is executed — that is `include`.

The distinction the API already makes should be mirrored in the sinks:

| Parameter | Correct sinks |
|---|---|
| view **name** (`view()`, `Factory::make()`, `first()`, `renderWhen()`, `renderUnless()`, `renderEach()`, `ResponseFactory::view()`) | `include` only |
| real filesystem **path** (`Factory::file($path)`, `ResponseFactory::download($file)`, `file($file)`) | `file` + `include` as today |

**Code to reproduce**

```php
function render(\Illuminate\Http\Request $request): \Illuminate\Contracts\View\View
{
    return view($request->input('template'));
}
```

Reported today:

```
TaintedFile: Detected tainted file handling
TaintedInclude: Detected tainted code passed to include or similar
```

Expected: `TaintedInclude` only.

Observed in the wild on `filamentphp/filament` at `packages/support/src/Components/ViewComponent.php:149`, where one `view($this->getView(), ...)` call produces both issues.

**Impacted Versions**

v4.15.4 / v3.15.4. Introduced by #1318, which added view-name sinks; the `file` kind was copied from `Factory::file()`, where it is correct.

**Additional context**

Sites to change — the `@psalm-taint-sink file` line on the name parameter only, keeping `include`:

- `stubs/common/Foundation/helpers.phpstub:405` (`view()` helper, `$view`)
- `stubs/common/View/Factory.phpstub:18` (`make()`), `:33` (`first()`, `$views`), `:39`, `:45`, `:54` + `:56` (`renderEach()`, `$view` and `$empty`)
- `stubs/common/Contracts/View/Factory.phpstub:35` (`make()`)
- `stubs/common/Routing/ResponseFactory.phpstub:64` and `stubs/common/Contracts/Routing/ResponseFactory.phpstub:40` (`view()`, `$view`)

Explicitly **not** changed: `Factory::file()` (`View/Factory.phpstub:24`, `Contracts/View/Factory.phpstub:19`) and the `download()`/`file()` response methods — those take real paths, so `file` belongs there.

No detection is lost by the change: no taint source declared in `stubs/` carries `file` without also carrying `include` (the `input` group alias, which covers `Request::input()` and friends, carries both). The one-line consequence is that a doubled report becomes a single accurate one.

`ContractStubSinkParityTest` enforces concrete/contract parity, so both halves of each pair must move together. Worth a `.phpt` pinning `view($tainted)` to exactly one `TaintedInclude`, and a negative pin keeping `Factory::file($tainted)` at both kinds.
